src/App.jsx

```diff
@@ -14,33 +15,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const isOriginalOrder = () => {
+    return JSON.stringify(goods) === JSON.stringify(goodsFromServer);
+  };
```

Comment:
The `isOriginalOrder` function compares the current `goods` array with `goodsFromServer` using `JSON.stringify`. This approach may not be reliable if the order of `goods` is changed and then reset, as it relies on string comparison. Consider using a different method to track the original order, such as a separate state variable or a more robust comparison logic.